packages/fxa-settings/README.md

```diff
@@ -9,6 +9,77 @@ This documentation is up to date as of 2020-08-03.
 - `yarn test` to run unit tests
 - `yarn storybook` to open Storybook
 
+### GQL and REST API Calls
+
+This package consumes the [FxA GraphQL API](https://github.com/mozilla/fxa/tree/main/packages/fxa-graphql-api) for the large majority of communication with the server. See the documentation to connect to the playground to view the API docs and schema.
+
+The application sends queries to the GQL server to request and receive data, and sends mutations for all other request types with aid of Apollo Client and Apollo Provider. See the [GQL documentation](https://graphql.org/learn/) to learn more.
+
+While most API calls can be performed with GQL, there are a few calls that must use REST to directly communicate with the auth server because these routes are not secured with a session token and the `graphql-server` sees Strings as a password type. To make calls to the auth server, use the function `useAuth()` [TO FINISH]
+
+### Global Application Data
+
+This application uses [Apollo client cache](https://www.apollographql.com/docs/react/caching/cache-configuration/) to store app-global state, holding both data received from the GQL server and local global state. You can see the shape of this data in the schema found within the `GetInitialState` query, noting that a `@client` directive denotes local data.
+
+Access the client cache data in top-level objects via `use` functions. At the time of writing, `useAccount` and `useSession` will allow you to access `data.account` and `data.session` respectively inside components where that data is needed. See the "Testing" portion of this doc for how to mock calls.
+
+#### `AlertBar` and `AlertExternal`
```

Comment:
Since there is a sizeable chunk of info here about AlertBar, could we store this in a README under `src/components/AlertBar` and link to it? This Readme is pretty beefy as it is, and I assume down the road we will have component-specific documentation.

Reply:
I think we talked about this in Slack and said to perhaps just leave it here - I only explain how to show and hide it which I don't foresee changing (it's the same way Payments has been revealing/hiding `DialogMessage`, and how we show/hide our `Modal`) and the `AlertBar` is such a core piece to familiarize yourself with if you're ramping up on this project since so much of what the user does triggers one. I think if we go the `Component/README.md` route it could be argued to move a lot of the chunks of this README could be broken up. I'll leave it for now, but we could revisit later maybe? I do agree this file is getting pretty long. But it's also nice having it all in one spot looking at https://github.com/mozilla/fxa/tree/issues/6192/packages/fxa-settings, maybe we need some links at the top...